core/src/main/java/org/bitcoinj/core/SegwitAddress.java

```diff
@@ -172,37 +176,33 @@ public String toString() {
      * @return constructed address
      * @throws AddressFormatException
      *             if something about the given bech32 address isn't right
-     * @deprecated Use {@link #fromBech32(Network, String)}
+     * @deprecated Use {@link AddressParser#parseAddress(String, Network)} or {@link AddressParser#parseAddressAnyNetwork(String)}
      */
     @Deprecated
     public static SegwitAddress fromBech32(@Nullable NetworkParameters params, String bech32)
             throws AddressFormatException {
-        return fromBech32(params != null ? params.network : null, bech32);
+        AddressParser parser = DefaultAddressParser.fromNetworks();
+        return (SegwitAddress) (params != null
+                ? parser.parseAddress(bech32, params.network())
+                : parser.parseAddressAnyNetwork(bech32)
```

Comment:
Similar here, could perhaps use `parseBech32AnyNetwork`?

Reply:
I responded above to the similar comment.

Reply:
Dito.